### ISSUE TITLE
Blast Furnaces and Distillation Units can all use the same 3x3x3 block of casings as other Blast Furnaces and Distillation Units

### ISSUE THREAD

**Issue by C1ffisme** [MikeTheShadow/VoluminousEnergy#33]
![2021-12-10_05 08 57](https://user-images.githubusercontent.com/9893266/145520708-eaaeef60-7733-4662-8ae3-fd410d04431d.png)

Yes, these are all valid multiblocks and they all work.

**Comment by Eelt** [comment]
Yeah, this is is an issue that I presume affects all versions of Voluminous Energy that has multiblocks. 
I probably won't rush a fix for this, but I will get around to fixing this eventually.
